test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -23,15 +30,29 @@ describe('Blueprints widget', () => {
     describe('for specific blueprint', () => {
         before(() => cy.uploadBlueprint('blueprints/simple.zip', emptyBlueprintName).refreshPage());
 
+        const editCopyInComposerButtonSelector = '[title="Edit a copy in Composer"]';
+
         it('should open Composer with the blueprint imported on "Edit a copy in Composer" icon click', () => {
-            // Click the action icon
-            getBlueprintRow(emptyBlueprintName).find('.external.share').click();
+            getBlueprintRow(emptyBlueprintName).find(editCopyInComposerButtonSelector).click();
 
             cy.window()
                 .its('open')
                 .should('be.calledWith', `/composer/import/default_tenant/${emptyBlueprintName}/blueprint.yaml`);
         });
 
+        it('should not show the "Edit a copy in Composer" button if it is turned off in the configuration', () => {
+            cy.editWidgetConfiguration('blueprints', () => {
+                cy.contains('Show the "Edit a copy in Composer" button')
+                    .parent('.field')
```

Comment:
Alternatively with one command:
```suggestion
                cy.contains('.field', 'Show the "Edit a copy in Composer" button')
```

Reply:
Good idea, done :+1: 

Hmm... I could have used GitHub's UI to commit, maybe it would not have lifted the existing approvals. My bad